scss/_print.scss

```diff
@@ -4,18 +4,54 @@
   .cads-error-summary,
   .cads-navigation,
   .cads-success-message,
-  .cads-related-content {
+  .cads-related-content,
+  .cads-skip-links {
     display: none !important;
   }
 
   // Components that adapt for printing
```

Comment:
Personal preference is to have a `@media print` block in the same file as the main component styles. But don't feel strongly about that, so this works too.

Reply:
Yeah I think what you're suggesting is a better approach

Reply:
going for a specific print query at the bottom of the file rather than dotted throughout